department-of-veterans-affairs/va.gov-cms
# VACMS-14817: Magichead Treelines

## Description

Relates to #14817 

## Testing done
In browser functional testing.

## Screenshots
![Screenshot 2023-09-12 at 2 29 05 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/384bf228-c2d6-4303-bec4-8f3b828465b0)
![Screenshot 2023-09-12 at 2 28 15 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/c1adc694-57cc-4eed-8636-ee15a70f65c8)
![Screenshot 2023-09-12 at 2 28 06 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/49340e07-4d12-41a7-b5cc-2ea911614a47)


## QA steps

- [ ] Ensure that the dashed lines in all the UI experiences look the like screenshot above.

- Log in as any user
- Go to Structure > Taxonomy. Select Benefit Detail Taxonomy. 
- Fill out required fields
- Click add Eligibility overview section
- Fill out the header and body fields and add a few more
- Save the form.
- Go back and edit the form. Click on "Show row weights".

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
